gettingStarted.html

```diff
@@ -44,156 +47,355 @@
         <div id="title">
             <h3 class="text-center page-title">Getting Started</h3>
         </div>
-        <div id="prerequisite" class="container-fluid">
-            <div class="row">
-                <h4 class="section-title col-md-11 offset-md-1 space">
-                    Prerequisites
-                </h4>
-            </div>
-            <div class="row">
-                <div class="col-md-5 offset-md-1 marg">
-                    <h5 class="device-title col-md-12">On Windows</h5>
-                    <div class="terminal col-md-12">
-                        <ul class="col-md-11">
-                            <li>Windows 10, 8.1, or 7</li>
+        <br />
+        <div class="tablist col-md-10 offset-md-1 centered" role="tablist">
+            <button
+                role="tab"
+                onclick="clickGetStartedTab('windows');"
```

Comment:
I don't think we should mix javascript and html. I think it would be better if we attached an event listener instead.

Reply:
updated